cli/command/run/cmd.go

```diff
@@ -234,18 +234,41 @@ func runCypress(cmd *cobra.Command, tc testcomposer.Client, rs resto.Client, as
 	rs.URL = regio.APIBaseURL()
 	as.URL = regio.APIBaseURL()
 
-	switch testEnv {
+	if testEnv != "" {
+		mode = testEnv
+	}
+
+	switch mode {
 	case "docker":
 		return runCypressInDocker(p, tc)
 	case "sauce":
 		return runCypressInSauce(p, regio, tc, rs, as)
+	case "":
+		dockerProject, sauceProject := cypress.SplitSuites(p)
+		// If there is no mode settings in config files, it should run all test suites in sauce by default.
+		if len(dockerProject.Suites) == 0 && len(sauceProject.Suites) == 0 {
```

Comment:
I find this mildly confusing 🤔 
If that statement is true, then
`dockerProject, sauceProject := cypress.SplitSuites(p)` does something unexpected. To have both be empty would be illegal. After all, I split my entire list of suites into these two buckets. How can the result of the split (!) amount to less items it total than it originally started with?